src/main/java/Task.java

```diff
@@ -0,0 +1,22 @@
+import java.util.*;
```

Comment:
Do not import everything using `*`, instead import only those classes that you need! You may want to prevent IntelliJ from optimizing the imports.